github_metrics/routes/api/turnaround.py

```
@@ -0,0 +1,467 @@
+                "avg_response_time_hours": float(row["avg_response_time_hours"] or 0),
+                "total_reviews": row["total_reviews"],
+                "repositories_reviewed": row["repositories"],
+            }
+            for row in by_reviewer_rows
+        ]
+
+        return {
+            "summary": summary,
+            "by_repository": by_repository,
+            "by_reviewer": by_reviewer,
+        }
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)